.argo/selenium_video_tool.yaml

```diff
@@ -37,7 +37,7 @@ args: [
 inputs:
   artifacts:
     VIDEO_INPUT:
-      path: /tmp
+      path: /tmp/video.flv
```

Comment:
Why do we need to change this path?
Full Argo workflows may only work with directories.

Reply:
Thank you for catching it! This is workaround for https://github.com/argoproj/argo/issues/310 . I've used another workaround, so workflow works in both argo and argo lite: argo-lite-deployment-2285963180-f9gbq